### PR TITLE
Footer

### LINKED ISSUE #4: Footer
Build Footer as per https://www.hubblehomes.com/ 

- [ ]  with links to live-site all content is imported
- [ ]  Build Mobile first
- [ ]  Tablet version
- [ ]  Desktop
Screenshot 2024-05-02 at 9 56 53 AM

Screenshot 2024-05-02 at 9 58 31 AM

### PR DESCRIPTION
**Pull request by dave-fink**
Minor adjustments from original
- cleaned up mobile breakpoints
- all fields in the subscribe form are required, it seemed odd to have the submit button above the name fields

Fix #4

Test URLs:
- Before: https://main--hubblehomes-com--aemsites.hlx.live/drafts/dfink/blank-test-page
- After: https://4-footer--hubblehomes-com--aemsites.hlx.live/drafts/dfink/blank-test-page
- After: https://4-footer--hubblehomes-com--aemsites.hlx.live/news/news-detail/2020-bca-fall-parade-of-homes-31

### CODE REVIEW

File: blocks/footer/footer.css [hunk 1]
```diff
@@ -1,16 +1,325 @@
...
+          font-size: .9em;
+          line-height: 1.5;
+          color: #555;
+          background: #fff;
+          border-radius: 4px;
+          border: none;
+          vertical-align: middle;
+          outline: none;
+          transition: all ease-in-out .15s;
+          &::placeholder { color: #555 }
```
Comment (bhellema) [inline blocks/footer/footer.css:141]: The hubblehomes.com text color for placeholders looks like #999.  
image

**Comment by bhellema** [comment]
The hubblehomes.com locations in the footer are organized by state > metro > city: 

image

It looks like we're just putting down a table with everything.

File: blocks/footer/footer.css (hunk 1 above)
Comment (dave-fink) [reply blocks/footer/footer.css:141]: Eagle eyes! :)

**Review by bhellema — APPROVED** [review]
Nice!

**Comment by codevoyage** [comment]
Comparison of Branch and Original
In comparing the branch and the original side by side, I noticed a few differences beyond those already mentioned:

1. At 1000 x 1205 px:

- Spacing Issue: There is a spacing issue between "Homeowner Referral" and "Our Locations." The columns in the original look skinnier.
- Text Styling: The "Stay in the know" text above the form appears to be bold or in a different font in the original.

 EDS:
Screenshot 2024-05-30 at 10 13 16 AM

Original: 
Screenshot 2024-05-30 at 10 16 29 AM


At 588 x 901 px:

- Link Layout: The original has its links in a single column, while EDS has them in two columns. In my opinion, two columns look better.
- Call to Action Banner: The original includes a call-to-action banner with contact links ("Call Us," "Leave a Message"), which is missing in the EDS version. This might be due to a different block not included in the footer.

 EDS:
Screenshot 2024-05-30 at 10 10 47 AM

Original:
Screenshot 2024-05-30 at 10 10 02 AM

**Comment by dave-fink** [comment]
@codevoyage Thank you for your detailed review, most of the items you pointed out have been addressed.  A few of the changes were intentional improvements from the original, as their mobile views tended to break down.  

I'll create a new ticket for the CTA - that will be it's own issue.

**Pull request merged**